torch/_dynamo/eval_frame.py

```diff
@@ -1531,7 +1531,9 @@ def patch():
         }
 
         excluded_single_tensor = {
-            radam,  # https://github.com/pytorch/pytorch/issues/117807
+            radam,  # https://github.com/pytorch/pytorch/issues/118230
+            adamax,  # https://github.com/pytorch/pytorch/issues/117836
+            asgd,  # https://github.com/pytorch/pytorch/issues/116052
```

Comment:
this should be removed if we re-enable the test above

Reply:
not for adamax tho, right?

Reply:
yup just ASGD only